src/modules/uv-searchfooterpanel-module/FooterPanel.ts

```diff
@@ -378,8 +378,9 @@ export class FooterPanel extends BaseFooterPanel {
         this.terms = terms;
 
         if (this.terms === '' || this.terms === this.content.enterKeyword) {
-            this.extension.showMessage(this.config.modules.genericDialogue.content.emptyValue, function(){
-                this.$searchText.focus();
+            const that = this;
+            this.extension.showMessage(this.extension.data.config.modules.genericDialogue.content.emptyValue, function(){
+                that.$searchText.focus();
```

Comment:
This could be simplified to:
```
() => {
    this.$searchText.focus();
}
```
avoiding the temporary variable

Reply:
Ok I updated the code. Didn't know about arrow functions